docs/kubernetes.adoc

```diff
@@ -47,43 +47,45 @@ For example **foo.bar.baz** will be converted to **FOO_BAR_BAZ**.
-| kubernetes.client.creator.class.name| Bool (true)    | Any | Fully qualified class name of a kubernetes client creator class (byon)           |
+| Option                              | Type           | Env | Description
+| kubernetes.master                   | URL            | Any | The URL to the Kubernetes master
+| kubernetes.domain                   | String         | OSE | Domain to use for creating routes for services
+| docker.registry                     | String         | Any | The docker registry
+| namespace.use.current               | Boolean (false)| Any | Don't generate a namespace use the current instead
+| namespace.use.existing              | String         | Any | Don't generate a namespace use the specified one instead
+| namespace.prefix                    | String (itest) | Any | If you don't specify a namespace, a random one will be created, with this prefix
+| namespace.lazy.enabled              | Bool (true)    | Any | Should the specified namespace be created if not exists, or throw exception?
+| namespace.destroy.enabled           | Bool (true)    | Any | Flag to destroy the namespace after the end of the test suite
+| namespace.destroy.confirm.enabled   | Bool (false)   | Any | Flag to ask for confirmation to delete the namespace
+| namespace.destroy.timeout           | Long           | Any | Time to wait before destroying the namespace
+| namespace.cleanup.enabled           | Bool (true)    | Any | Flag to clean (delete resources) the namespace after the end of the test suite
+| namespace.cleanup.confirm.enabled   | Bool (false)   | Any | Flag to ask for confirmation to clean the namespace
+| namespace.cleanup.timeout           | Long           | Any | Time to wait when cleaning up the namespace
+| env.init.enabled                    | Bool (true)    | Any | Flag to initialize the environment (apply kubernetes resources)
+| env.config.url                      | URL            | Any | URL to the Kubernetes JSON/YAML (defaults to classpath resource kubernetes.json)
+| env.config.resource.name            | String         | Any | Option to select a different classpath resource (other than kubernetes.json)
+| env.script.env                      | String         | Any | Key/Value pairs to pass to setup scripts as environment variables
+| env.setup.script.url                | URL            | Any | Option to select a shell script that will setup the environment
+| env.teardown.script.url             | URL            | Any | Option to select a shell script to tear down / cleanup the environment
+| env.dependencies                    | List           | Any | Whitespace separated list of URLs to more dependency kubernetes.json
+| env.console.logs.copy               | Bool (false)   | Any | Whether to capture the pods logs and save them into the filesystem - as individual files, one for each pod. Filenames will be "ClassName-[MethodName-]-PodName.log".
```

Comment:
As per my understanding, we are capturing pod logs for entire execution of test suite. If so then do we really need classname & methodname for pod logs `fileName`? Only pods name is not enough here?

correct me if I am wrong

Reply:
I container mode is set to class instead of suite then we need classname. I don't think we need method.

Reply:
Yeah, the idea is to easily identify which tests/methods created which pods. So, if you have lots (think of hundreds, thousands) of tests and just a few of them are failing, you want to check the logs of the pods created by that failing test, instead of digging into a huge log file.